README.md

```diff
@@ -106,7 +106,7 @@ python dummy.py \
 #### `keras_house_prices`
 
 **All the commands in this section are run from the
-`python/client_examples/keras_house_prices` directory.**
+python/client_examples/keras_house_prices directory.**
```

Comment:
i would suggest to keep the inline code highlighting. since combined markdown expressions seem to be implossible with `m2r`, we could do it this way:
```
**All the commands in this section are run from the** `python/client_examples/keras_house_prices` **directory.**
```

Reply:
This wouldn't be a problem anymore with https://github.com/xainag/xain-fl/pull/362